nrfx/drivers/src/nrfx_rramc.c

```diff
@@ -66,7 +66,7 @@ NRFX_STATIC_INLINE uint32_t total_memory_size_get(void)
     return (size * 1024UL);
 }
 
-NRFX_STATIC_INLINE bool is_valid_address(uint32_t addr, bool uicr_allowed)
+NRFX_STATIC_INLINE __attribute__((unused)) bool is_valid_address(uint32_t addr, bool uicr_allowed)
```

Comment:
nrfx seems to prefer `__UNUSED` defined by MDK: https://github.com/zephyrproject-rtos/hal_nordic/blob/37c38a471c042d890180cfb79cf788a32a006b80/nrfx/drivers/src/nrfx_spim.c#L122
but I guess it's okay to go with the attribute directly for now.

Reply:
Yes, since this is an urgent patch I will rather not complicate things for now.